### PR TITLE
Increase the tolerance

### PR DESCRIPTION
**Pull request by gautamkmr**
@piiswrong @nswamy @cjolivier01

### CODE REVIEW

File: tests/python/unittest/test_operator.py [hunk 1]
```diff
@@ -953,16 +953,16 @@ def check_batchnorm_training(stype):
...
-                check_numeric_gradient(test, in_location, xmean_std, numeric_eps=1e-2, rtol=0.2, atol=0.01)
+                check_numeric_gradient(test, in_location, xmean_std, numeric_eps=1e-2, rtol=0.2, atol=1e-4)
 
                 test = mx.symbol.BatchNorm(data, fix_gamma=False, axis=chaxis)
-                check_numeric_gradient(test, in_location, xmean_std, numeric_eps=1e-2, rtol=0.2, atol=0.01)
+                check_numeric_gradient(test, in_location, xmean_std, numeric_eps=1e-2, rtol=0.2, atol=1e-4)
 
                 test = mx.symbol.BatchNorm(data, fix_gamma=False, use_global_stats=True, axis=chaxis)
-                check_numeric_gradient(test, in_location, xmean_std, numeric_eps=1e-2, rtol=0.2, atol=0.01)
+                check_numeric_gradient(test, in_location, xmean_std, numeric_eps=1e-2, rtol=0.2, atol=1e-4)
```
Comment (szha) [inline tests/python/unittest/test_operator.py]: This looks like a decrease to me.

**Comment by cjolivier01** [comment]
Is this newly failing? That's a pretty bad tolerance. Are the c++ unit tests being run? Do they pass?

File: tests/python/unittest/test_operator.py (hunk 1 above)
Comment (cjolivier01) [reply tests/python/unittest/test_operator.py]: Yeah it is an increase :)

**Comment by gautamkmr** [comment]
https://builds.apache.org/blue/organizations/jenkins/incubator-mxnet/detail/master/390/pipeline/
https://builds.apache.org/blue/organizations/jenkins/incubator-mxnet/detail/master/391/pipeline/
are two examples of failure.

**Comment by cjolivier01** [comment]
What I mean is, did it just recently start failing or it's been failing periodically? I didn't see it on the list of failing tests before. What's the failure rate now?

Are the cpp unit tests running? Are they passing? Batch norm operator is tested exhaustingly in the cpp unit tests.

**Comment by gautamkmr** [comment]
@cjolivier01  I have seen this failing since last week >75%,  we are not running CPP test :(

**Comment by cjolivier01** [comment]
Was it failing before last week? Because if it wans t, it's not the fault
of the test. Not falling without 0.01 error is a problem.

**Comment by gautamkmr** [comment]
@cjolivier01   I saw the failure since last two weeks, haven't noticed before.

**Comment by gautamkmr** [comment]
The test has been failing continuously  here is another instance today 
https://builds.apache.org/blue/organizations/jenkins/incubator-mxnet/detail/master/395/pipeline

**Comment by gautamkmr** [comment]
Whops wrong test fix.

**Pull request closed without merging**